components/idp.js

```diff
@@ -86,6 +85,20 @@ module.config(function($routeProvider, routeResolverProvider) {
     })
     .when('/join', {
       title: 'Create Identity',
+      /* @ngInject */
+      resolve: {loggedIn: function($location, brSessionService, config) {
```

Comment:
Move up to where resolve function is to ensure ng-annotation works properly. Change "loggedIn" to "redirectIfReferred" or something.